drivers/spi/Kconfig.mcux_dspi

```diff
@@ -1,11 +1,37 @@
 # MCUXpresso SDK SPI
 
 # Copyright (c) 2016, Freescale Semiconductor, Inc.
-# Copyright (c) 2017, NXP
+# Copyright (c) 2017, 2021, NXP
```

Comment:
Remove the duplicate `Signed-off-by: Hake Huang <hake.huang@oss.nxp.com>` in commit `drivers: spi: Update mcux driver to support edma`

Reply:
ok

Reply:
ok